### PR TITLE
My pull requestes

### PR DESCRIPTION
**Pull request by WALWALASAFI**
Students survey analysis for feedback and insights.

### CODE REVIEW

**Comment by ahzamir** [comment]
Hey @WALWALASAFI 
- You have some linters issues, always solve your linters and then submit your project for review.
- I can not provide you feedback till you don't solve the errors.

**Comment by WALWALASAFI** [comment]
ok, sir

**Comment by WALWALASAFI** [comment]
I have fixed the linter dear sir,

**Comment by WALWALASAFI** [comment]
i have opened it sir

**Comment by WALWALASAFI** [comment]
@ahzamir

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @WALWALASAFI 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Copy your HTML file code and paste it to the freecodecamp website survey form project to see which requirements your project is missing solve those missing requirements and mention me here using @ahzamir to re-review your project.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
❌ You have followed and completed the Survey Form and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

File: index.html
Comment (ahzamir) [inline index.html:1]: Your codes are not alligned correctly and are not clean try to make their spacing right.
- [**Suggession**] - On your HTML file, at the bottom right side of your vscode click on `Spaces`  then click on `Index using spaces`, and then select `2` from there. Then align your components according to the 2 spaces.
- Example:
```
<div>
  <p>...</p>
  <h1>.... </h1>
  <img ... />
</div>
```

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @WALWALASAFI 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Check the Inline comments below.
- Copy your HTML file code and paste it to the freecodecamp website survey form project to see which requirements your project is missing solve those missing requirements and mention me here using @ahzamir to re-review your project.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
❌ You have followed and completed the Survey Form and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by WALWALASAFI** [comment]
Hi dear sir, 
i hope ur fine and doing great.
the code have been checked and accepted in free code camp 
also, it does not have any linter error.
finally is there anything remind for approving of the project.
kindly waiting for the feedback to make it correct as early as i can.

**Comment by WALWALASAFI** [comment]
Hi dear sir,
i hope ur fine and doing great.
the code have been checked and accepted in free code camp
also, it does not have any linter error.
finally is there anything remind for approving of the project.
kindly waiting for the feedback to make it correct as early as i can.
Uploading survey form.png…
@ahzamir 
@ahzia

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Survey Form</title>
+<link rel="stylesheet" href=".css">
+</head>
```
Comment (ahzamir) [inline index.html]: - Clean your code for this part

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+<body>
+<h1 id="title">Survey Form</h1>
+<p id="description">Please take a moment to fill out our survey-form.</p>
+<form id="survey-form">
```
Comment (ahzamir) [inline index.html]: - Clean here

File: index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  </div>
+  <form id="survey-form">
+  <label for="checkbox1">
+    <input type="checkbox" id="checkbox1" name="option1" value="value1"> Option 1
+  </label>
+  <label for="checkbox2">
+    <input type="checkbox" id="checkbox2" name="option2" value="value2"> Option 2
+  </label>
+  <button type="submit" id="submit">Submit</button>
+</form>
```
Comment (ahzamir) [inline index.html]: - Here is clean just all of them should be two spaces ahead.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    <input type="radio" id="radio1" name="radio" value="option1">
+    <label for="radio1">Option 1</label><br>
+    <input type="radio" id="radio2" name="radio" value="option2">
+    <label for="radio2">Option 2</label><br>
+  </div>
+  <div>
+    <label for="comments">Additional Comments:</label><br>
+    <textarea id="comments" name="comments" rows="4" placeholder="Enter your comments"></textarea>
+  </div>
+  <form id="survey-form">
```
Comment (ahzamir) [inline index.html]: - Remove this

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @WALWALASAFI 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Although your code passes all the checks on freecodecamp website, but your HTML code is not clean please check the comments below for more details.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Survey Form and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
❌ Your code is clean

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by WALWALASAFI** [comment]
hi sir, 
i hope ur fine .
i did whatever u said , i cleaned unneccessary code and give 2 space ahead.
@ahzamir

**Review by ahzamir — CHANGES_REQUESTED** [review]
@WALWALASAFI 

- Still your code spacing is not right, try to get help from your partner.

**Comment by WALWALASAFI** [comment]
Hi dear sir, 
i give space correctly. trust me sir i asked from my partner but she doesnt have any idea. so i looked up and tried much.
inshallah the code  will be correct and cleaned in this time.
@ahzamir

**Comment by ahzamir** [comment]
> Hi dear sir, i give space correctly. trust me sir i asked from my partner but she doesnt have any idea. so i looked up and tried much. inshallah the code will be correct and cleaned in this time. @ahzamir

- Well done now it's correct.

**Review by ahzamir — APPROVED** [review]
### Hi @WALWALASAFI 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Survey Form project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by WALWALASAFI** [comment]
thanks a bunch dear sir🙏🙏💜

**Pull request merged**